polkadot/runtime/parachains/src/builder.rs

```diff
@@ -442,7 +449,7 @@ impl<T: paras_inherent::Config> BenchBuilder<T> {
 		self.validators = Some(validators_shuffled);
 		self.block_number = block_number;
 		self.session = target_session;
-		assert_eq!(paras::Pallet::<T>::parachains().len(), total_cores as usize);
+		// assert_eq!(paras::Pallet::<T>::parachains().len(), total_cores as usize);
```

Comment:
this needs to be removed, right?

Reply:
yeah, but maybe it can still be checked if there are no elastic paras.

Reply:
added back the correct check accounting for extra cores from elastic paras.